active-directory-b2c-wpf/MainWindow.xaml.cs

```diff
@@ -29,7 +29,8 @@ private async void SignInButton_Click(object sender, RoutedEventArgs e)
             IEnumerable<IAccount> accounts = await App.PublicClientApp.GetAccountsAsync();
             try
             {
-                authResult = await (app as PublicClientApplication).AcquireTokenInteractive(App.ApiScopes, null)
+                authResult = await (app as PublicClientApplication).AcquireTokenInteractive(App.ApiScopes)
+                    .WithParentActivityOrWindow(this)
```

Comment:
Hmm, I'm not sure this will work. I mean the auth will work but the browser window will not be correctly centered around the WPF window. 

If you upgrade to not using package.config, then you'll only see the overloads that take IWin32 and IntPtr.

The correct way to pass a window handle from WPF is 

```csharp
 .WithParentActivityOrWindow(new WindowInteropHelper(this).Handle)
```

Reply:
I suppose we could make this a bit easier for developers when they use .WithParentActivityOrWindow(object window)

1. If they pass null -> do nothing
2. If they pass an object -> make sure it's the right type

Reply:
Thanks Bogdan
https://github.com/Azure-Samples/active-directory-b2c-dotnet-desktop/pull/23/commits/41c70b63f996623b03053c622530bbcd4b927a73